frontend/containers/open-call-page/funding-impact/component.tsx

```diff
@@ -0,0 +1,126 @@
+import { FC } from 'react';
+
+import { FormattedMessage } from 'react-intl';
+
+import { useRouter } from 'next/router';
+
+import SDGs from 'containers/sdgs/component';
+
+import LayoutContainer from 'components/layout-container';
+
+import { OpenCallFundingImpactProps } from '.';
+
+export const OpenCallFundingImpact: FC<OpenCallFundingImpactProps> = ({
+  openCall,
+  allSdgs,
+  instrumentTypes,
+  fundingRef,
+  impactRef,
+}) => {
+  const { locale } = useRouter();
+
+  const {
+    impact_description,
+    sdgs,
+    maximum_funding_per_project,
+    funding_priorities,
+    funding_exclusions,
+  } = openCall;
+
+  const openCallSdgs = allSdgs.filter((sdg) => sdgs.includes(Number(sdg.id)));
+
+  return (
+    <div>
+      <div ref={fundingRef}>
+        <LayoutContainer>
+          <LayoutContainer>
+            <h2 className="mb-10 font-serif text-4xl font-bold">
+              <FormattedMessage defaultMessage="Funding information" id="mEYG82" />
+            </h2>
+          </LayoutContainer>
+          <LayoutContainer className="text-white px-15 lg:px-15 bg-green-dark py-14 rounded-3xl">
+            <div>
+              <h3 className="font-serif text-3xl font-semibold">
+                <FormattedMessage defaultMessage="Funding available per project" id="+USuwC" />
+              </h3>
+              <div className="flex flex-col mt-10 gap-14 sm:flex-row">
```

Comment:
The spacing is smaller on the design:
![The space between maximum value and instrument types is narrower than the titles below](https://user-images.githubusercontent.com/6073968/185359197-72d3e99d-da33-4e39-816f-efb5a16bcdd5.png)